.github/workflows/docs.yml

```diff
@@ -31,6 +31,7 @@ jobs:
           fail: true
       - name: Rust Cache
         uses: Swatinem/rust-cache@v2
+        timeout-minutes: 5
```

Comment:
`continue-on-error` here?

Reply:
good catch, I forgot to save the file :crying_cat_face: